### ISSUE TITLE
barplot with monthly cost

### ISSUE THREAD

**Issue by ChrisZenhub** [architecture-building-systems/hive#305]
for levelized CAPEX and OPEX

**Comment by amraladdin** [comment]
Update: From the Visualization design perspective, this issue connects to issues #304 #306 and #307. Will be discussed on July 02 meeting.

**Comment by amraladdin** [comment]
![image](https://user-images.githubusercontent.com/16256867/86484734-2c39a080-bd57-11ea-920e-690902a3c5bc.png)


![image](https://user-images.githubusercontent.com/16256867/86484774-41163400-bd57-11ea-9b73-f9d696056845.png)

**Comment by daren-thomas** [comment]
mostly implemented, see PR #336

**Comment by ChrisZenhub** [comment]
would you like to keep this issue for the breakdown of costs? If not, feel free to close it.

**Comment by amraladdin** [comment]
You're asking @daren-thomas , right?

**Comment by daren-thomas** [comment]
@amraladdin yes, I think he is. @christophwaibel yes, let's keep it until it's resolved.
